test/test_monitor.py

```diff
@@ -79,6 +80,17 @@ def test_cleanup_executors_on_client_close(self):
         for executor in executors:
             wait_until(lambda: executor._stopped, f"closed executor: {executor._name}", timeout=5)
 
+    @unittest.skipIf(sys.version_info[:2] >= (3, 12), reason="Python version must be (>=3.12)")
+    def test_no_thread_start_runtime_err_on_shutdown(self):
+        """Test we silence noisy runtime errors fired when the MongoClient spawns a new thread
+        on process shutdown."""
+        command = ["python", "-c", "'from pymongo import MongoClient; c = MongoClient()'"]
+        completed_process: subprocess.CompletedProcess = subprocess.run(
+            " ".join(command), shell=True, capture_output=True
+        )
+
+        assert not completed_process.stderr
```

Comment:
Let's assert stdout is empty too? Also should this be `self.assertEmpty` or `self.assertFalse`?

Reply:
I ended up using `self.assertFalse`